frontend-react/src/pages/dashboard/Dashboard.tsx

```diff
@@ -3,14 +3,52 @@ import { Helmet } from "react-helmet-async";
 import React from "react";
 
 import { FeatureName } from "../../AppRouter";
+import { AuthElement } from "../../components/AuthElement";
+import { MemberType } from "../../hooks/UseOktaMemberships";
+import { USNavLink } from "../../components/USLink";
+import { withCatchAndSuspense } from "../../components/RSErrorBoundary";
 
-export function Dashboard() {
+import DashboardTable from "./DashboardTable/DashboardTable";
+
+function Dashboard() {
```

Comment:
We should align the name of this component with its name representation in CSS (either call this DataDashboard or change the css class name to rs-dashboard).

Reply:
@jpandersen87 should I rename the file as well, or is Dashboard ok?